nixos/maintainers/scripts/oci/create-image.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eu
+
+export NIX_PATH=nixpkgs=$(dirname $(readlink -f $0))/../../../..
+export NIXOS_CONFIG=$(dirname $(readlink -f $0))/../../../modules/virtualisation/oci-image.nix
+
+nix-build '<nixpkgs/nixos>' \
+   -A config.system.build.OCIImage \
+   --argstr system x86_64-linux \
+   --option system-features kvm \
+   -o oci-image \
+   -j 8
```

Comment:
```suggestion
   -j$NIX_BUILD_CORES
```
Can this be used here?

Reply:
It seems that NIX_BUILD_CORES is only available inside certain nix environments such as inside a nix-shell but not outside of those environments. I think it would be better to set the maximum number of jobs to auto, which should achieve the same result without relying on the environment.

Reply:
The reason we are normally not doing this is that if you have 32 cores you might not want to use them all for one build.

Reply:
Alright, should I try to read NIX_BUILD_CORES if it exists and fall back to a constant (e.g. 8) instead? Or should I omit the option altogether to fall back to the option defined in nix.conf?